githubbot/githubbot/handler.go

```diff
@@ -212,7 +213,7 @@ func (h *Handler) handleWatch(cmd string, msg chat1.MsgSummary, create bool, cli
 		if err != nil {
 			return fmt.Errorf("error getting subscription: %s", err)
 		}
-		_, err := h.kbc.SendMessageByConvID(msg.ConvID, fmt.Sprintf("You aren't subscribed to notifications for %s!", args[0]))
+		_, err := h.kbc.SendMessageByConvID(msg.ConvID, fmt.Sprintf("You aren't subscribed to updates yet!\nFirst do this: `!github subscribe %s`", args[0]))
```

Comment:
This shouldn't tell you to subscribe to updates first if you're running `!unsubscribe`, I think. Also, could you change "First do this" to "Send this first"?

Reply:
I've added an if statement to change the message (between sub and unsub), if it's too messy we can just go back to the original code. But thought it might be easier for the user if we showed them what to do to solve the error